Matrix_Operations/Java/Matrix_transpose.java

```diff
@@ -0,0 +1,63 @@
+// Java Program to calculation transpose of a matrix (2D Array)
+import java.util.Scanner;
+
+public class Matrix_transpose {
+
+	// Find transpose of matrix
+	public static void transpose_matrices(int[][] matrix) {
+		
+		int index1, index2;
+		int[][] matrix_ans = new int[matrix.length][matrix[0].length];
```

Comment:
Please remove extra spaces. Only 4 space indentation required from start of a new block.